### PR TITLE
Improves Swift compatibility

### PR DESCRIPTION
**Pull request by shagedorn**
Verified that it works as expected in Trails.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
   * Example updated to work with CocoaPods 1.0
   * Private headers are marked private in Podspec
   * Adds subspec for Health support
+* Improves Swift support by adding nullability annotations
```
Comment (felixLam) [inline CHANGELOG.md]: Indendation

File: ISHPermissionKit/ISHPermissionsViewController.h [hunk 2]
```diff
@@ -112,6 +112,7 @@ typedef void (^ISHPermissionsViewControllerCompletionBlock)(void);
 - (ISHPermissionRequestViewController *)permissionsViewController:(ISHPermissionsViewController *)vc requestViewControllerForCategory:(ISHPermissionCategory)category;
```
Comment (felixLam) [inline ISHPermissionKit/ISHPermissionsViewController.h]: Return value is `nullable`

File: ISHPermissionKit/Requests/ISHPermissionRequestNotificationsLocal.h [hunk 3]
```diff
@@ -15,10 +15,12 @@
...
  */
 @interface ISHPermissionRequestNotificationsLocal : ISHPermissionRequest
+
 /**
  *  Settings to configure the notification.
  *
  *  @sa ISHPermissionCategoryNotificationLocal
  */
-@property (nonatomic) UIUserNotificationSettings *noticationSettings;
+@property (nonatomic, nonnull) UIUserNotificationSettings *notificationSettings;
```
Comment (shagedorn) [inline ISHPermissionKit/Requests/ISHPermissionRequestNotificationsLocal.h]: This obviously breaks backwards compatibility, but it's a trivial fix.

I don't think it either justifies a 2.0 release nor adding a separate property and deprecating the old, which would have the potential of new bugs...

File: CHANGELOG.md (hunk 1 above)
Comment (shagedorn) [inline CHANGELOG.md]: It's a new bullet point whereas the others are sub-points of one separate bullet point

File: ISHPermissionKit/Requests/ISHPermissionRequestNotificationsLocal.h (hunk 3 above)
Comment (felixLam) [inline ISHPermissionKit/Requests/ISHPermissionRequestNotificationsLocal.h]: agreed.

File: ISHPermissionKit/ISHPermissionsViewController.h (hunk 2 above)
Comment (shagedorn) [inline ISHPermissionKit/ISHPermissionsViewController.h]: No. The data source is required to provide a controller. There's even an assert in the code.

Comment (felixLam) [inline ISHPermissionKit/ISHPermissionsViewController.h]: Sorry. I was confused by the diff.

**Comment by felixLam** [comment]
:shipit:

**Pull request merged**